test/unit/api/python/test_solver.py

```diff
@@ -84,6 +84,33 @@ def test_mk_bit_vector_sort(solver):
     with pytest.raises(RuntimeError):
         solver.mkBitVectorSort(0)
 
+def test_mk_finite_field_sort(solver):
```

Comment:
This is missing a check for `solver.mkFiniteFieldSort` with an `int` and a base other than `10` (which should raise a RuntimeError.